### ISSUE TITLE
General enhancements to the data model within API 2.0

### ISSUE THREAD

**Issue by acka47** [hbz/lobid-resources#5]
This issue is to collect possible enhancements to the lobid data model we could implement with API 2.0.

**Comment by acka47** [comment]
**wdrs:describedby and foaf:primaryTopicOf**

Currently, we use both these very similar properties, e.g.:

```javascript
{
  "@context" : "http://lobid.org/context/lobid-resources.json",
  "@id" : "http://lobid.org/resource/HT003160768",
  "describedby" : [ {
    "@id" : "http://lobid.org/resource/HT003160768/about",
    "dateCreated" : "19931111",
    "dateModified" : "20031104"
  } ],
  "isPrimaryTopicOf" : [ {
    "@id" : "http://193.30.112.134/F/?func=find-c&ccl_term=IDN%3DHT003160768"
  } ]
}
```

We might think about using only one and putting the linked objects in an array:

```javascript
{
  "@context" : "http://lobid.org/context/lobid-resources.json",
  "@id" : "http://lobid.org/resource/HT003160768",
  "isPrimaryTopicOf" : [ {
    "@id" : "http://lobid.org/resource/HT003160768/about",
    "dateCreated" : "19931111",
    "dateModified" : "20031104"
    },
    {
      "@id" : "http://193.30.112.134/F/?func=find-c&ccl_term=IDN%3DHT003160768"
    }
   ]
}
```

**Comment by acka47** [comment]
**Use bibo:isbn as ISBN property**

We should remember switching to the general ISB property as we don't differ between ISBN-10 and ISBN-13 in our data. (Currently, we use ''bibo:isbn13'' for both.)

**Comment by acka47** [comment]
Some time ago we chose `umbel:isLike` to add a URN link to a resource as `owl:sameAs` might not be correct when the link goes to a landing page. We might now use `schema:sameAs` instead of `umbel:isLike` as the vocabulary is more common.

**Comment by acka47** [comment]
From https://github.com/hbz/lobid-resources/issues/108#issuecomment-266028522:

> I tried different variants on going through all properties and types to see what vocabulary options we have. The best way seems to be to comment the context file as it contains all classes and properties used (but not the `medium` values, will also have to check this.) I did this in an hjson file, see https://gist.github.com/acka47/74f24ec7a23a152f30575d64fb23fa44. I would like to put this somewhere in this repo. Where is the best place? Shall I just put it in the root directory?

/cc @dr0i

**Comment by acka47** [comment]
As far as I can see, we will be able to remove at least 38 entries from the JSON-LD context, see https://gist.github.com/acka47/74f24ec7a23a152f30575d64fb23fa44 (search for "removed").

**Comment by acka47** [comment]
@ChristophEwertowski Let's go through https://gist.github.com/acka47/74f24ec7a23a152f30575d64fb23fa44 for last adjustments on Thursday.

**Comment by ChristophEwertowski** [comment]
@acka47 Went through the annotated context on a fork of your file: https://gist.github.com/ChristophEwertowski/d4daa9b31390e00ec7cc1125e36347b5

I added possible alternatives and looked up which properties and classes aren't used anymore. Some questions and comments were also added. In general I ordered the options from the one which is most preferable to the least preferable (in my point of view at least and even then there are some cases which are not so easy to decide).
